sdk/storage/Azure.Storage.Common/tests/Shared/TestConfigurationsTemplate.xml

```diff
@@ -104,5 +105,23 @@
       <FileServiceSecondaryEndpoint>http://[NAMESPACE-ACCOUNT]-secondary.file.core.windows.net</FileServiceSecondaryEndpoint>
       <TableServiceSecondaryEndpoint>http://[NAMESPACE-ACCOUNT]-secondary.table.core.windows.net</TableServiceSecondaryEndpoint>
     </TenantConfiguration>
+
+    <!-- A managed disk storge account -->
```

Comment:
nit: typo

Also, how does this relate to the page range changes?

Reply:
Fixed.

The new header only works with a managed disk accounts.

Reply:
Should we mention this in the doc comments for the parameter?